lte/gateway/docker/services/python/Dockerfile

```diff
@@ -69,6 +72,28 @@ RUN mkdir -p ${SWAGGER_CODEGEN_DIR}; \
   wget --no-verbose https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/${CODEGEN_VERSION}/swagger-codegen-cli-${CODEGEN_VERSION}.jar -O ${SWAGGER_CODEGEN_JAR}
 
 WORKDIR /magma/lte/gateway/python
+COPY \
+    ./lte/gateway/python/Makefile \
+    ./lte/gateway/python/defs.mk \
+    ./lte/gateway/python/setup.py \
+    $MAGMA_ROOT/lte/gateway/python/
+COPY ./lte/gateway/python/load_tests $MAGMA_ROOT/lte/gateway/python/load_tests
+COPY ./lte/gateway/python/scripts $MAGMA_ROOT/lte/gateway/python/scripts
+COPY ./orc8r/gateway/python/magma $MAGMA_ROOT/orc8r/gateway/python/magma
+COPY \
```

Comment:
you probably want to include the entire orc8r/gateway/python part

Reply:
In my testing it was enough to just include the service code base (at least for the `test_attach_detach` test to work). But maybe some or all of the scripts would also be relevant. I don't think the makefile etc are needed in the containers

Reply:
We could do the same for the `lte/gateway/python` folder, but we do not need the `integ_test` folder here (we could delete it after copying, which would make the Dockerfile at least more readable)